src/schema/settings.ts

```diff
@@ -123,6 +130,11 @@ export const typeDefs = gql`
     Whether to open articles on new tab
     """
     openNewTab: Boolean
+
+    """
+    Whether to show the sidebar open
+    """
+    openSidebar: Boolean!
```

Comment:
This shouldn't be required just like the others.

```suggestion
    openSidebar: Boolean
```